src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse42.cs

```diff
@@ -18,115 +18,242 @@ public static class Sse42
 
         /// <summary>
         /// int _mm_cmpistra (__m128i a, __m128i b, const int imm8)
         ///   PCMPISTRI xmm, xmm/m128, imm8
+        /// </summary>
+        public static bool CompareImplicitLengthNotCAndNotZ(Vector128<ushort> left, Vector128<ushort> right, StringComparisonMode mode) => CompareImplicitLengthNotCAndNotZ(left, right, mode);
+
+        /// <summary>
         /// int _mm_cmpistrc (__m128i a, __m128i b, const int imm8)
         ///   PCMPISTRI xmm, xmm/m128, imm8
-        /// int _mm_cmpistro (__m128i a, __m128i b, const int imm8)
-        ///   PCMPISTRI xmm, xmm/m128, imm8
-        /// int _mm_cmpistrs (__m128i a, __m128i b, const int imm8)
+        /// </summary>
+        public static bool CompareImplicitLengthC(Vector128<sbyte> left, Vector128<sbyte> right, StringComparisonMode mode) => CompareImplicitLengthC(left, right, mode);
+
+        /// <summary>
+        /// int _mm_cmpistra (__m128i a, __m128i b, const int imm8)
         ///   PCMPISTRI xmm, xmm/m128, imm8
-        /// int _mm_cmpistrz (__m128i a, __m128i b, const int imm8)
+        /// </summary>
+        public static bool CompareImplicitLengthC(Vector128<byte> left, Vector128<byte> right, StringComparisonMode mode) => CompareImplicitLengthC(left, right, mode);
+
+        /// <summary>
+        /// int _mm_cmpistra (__m128i a, __m128i b, const int imm8)
         ///   PCMPISTRI xmm, xmm/m128, imm8
         /// </summary>
-        public static bool CompareImplicitLength(Vector128<byte> left, Vector128<byte> right, ResultsFlag flag, StringComparisonMode mode) => CompareImplicitLength(left, right, flag, mode);
+        public static bool CompareImplicitLengthC(Vector128<short> left, Vector128<short> right, StringComparisonMode mode) => CompareImplicitLengthC(left, right, mode);
 
         /// <summary>
         /// int _mm_cmpistra (__m128i a, __m128i b, const int imm8)
         ///   PCMPISTRI xmm, xmm/m128, imm8
-        /// int _mm_cmpistrc (__m128i a, __m128i b, const int imm8)
+        /// </summary>
+        public static bool CompareImplicitLengthC(Vector128<ushort> left, Vector128<ushort> right, StringComparisonMode mode) => CompareImplicitLengthC(left, right, mode);
+
+        /// <summary>
+        /// int _mm_cmpistro (__m128i a, __m128i b, const int imm8)
+        ///   PCMPISTRI xmm, xmm/m128, imm8
+        /// </summary>
+        public static bool CompareImplicitLengthO(Vector128<sbyte> left, Vector128<sbyte> right, StringComparisonMode mode) => CompareImplicitLengthO(left, right, mode);
+
+        /// <summary>
+        /// int _mm_cmpistro (__m128i a, __m128i b, const int imm8)
+        ///   PCMPISTRI xmm, xmm/m128, imm8
+        /// </summary>
+        public static bool CompareImplicitLengthO(Vector128<byte> left, Vector128<byte> right, StringComparisonMode mode) => CompareImplicitLengthO(left, right, mode);
```

Comment:
What does `O` mean?  Similiarly, what does the `S` suffix mean below?  I agree with @4creators that we typically don't use single letter abbreviations for prefixes/suffixes in names.